PatternPal/PatternPal.Core/Checks/ParameterCheck.cs

```diff
@@ -16,6 +28,104 @@ internal class ParameterCheck : CheckBase
         RecognizerContext ctx,
         INode node)
     {
-        throw new NotImplementedException();
+        IParameterized hasParameters = CheckHelper.ConvertNodeElseThrow<IParameterized>(node);
+
+        // Retrieve the parameters the method has to compare to list of TypeChecks and convert to
```

Comment:
It is not necessarily a method. It can be a constructor as well. So use something like entity instead to refer to the node